target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ board_config_update
 board=$(board_name)
 
 case "$board" in
+"checkpoint,l-50")
+	ucidef_set_interface_lan "lan1 lan2 lan3 lan4 lan5 lan6 lan7 lan8 dmz"
+	ucidef_set_interface_wan "eth0"
```

Comment:
Based on other targets I'm used to, I'd choose ucidef_set_interfaces_lan_wan here.
But should be cosmetical.